CORE/world_gadgets/router/page_spec.rb

```diff
@@ -33,7 +24,7 @@ def inherit_details(parent_page)
   end
 
   def add_route(target_page, action, prerequisites)
-    @routes[target_page] ||= Route.new( target_page, action, prerequisites )
+    @routes[target_page] ||= Route.new( target_page, action, [ prerequisites ].flatten )
```

Comment:
I'm having a slight amount of trouble following this. Why does it need to be flattened now?